e2e/createAccount/createAccount.test.js

```diff
@@ -1,53 +1,145 @@
 /* eslint-disable flowtype/require-valid-file-annotation */
 /* globals describe beforeEach expect it element by */
-
-import { launchAppWithPermissions, navigateToHome } from '../utils.js'
+/* eslint-env detox/detox, jest */
+
+import { Date } from 'core-js'
+import { waitFor } from 'detox'
+
+import { launchAppWithPermissions } from '../utils.js'
+
+// FUNCTIONS
+// const sleep = milliseconds => new Promise(resolve => setTimeout(resolve, milliseconds))
+const genUsername = () => 'TU' + Date.now()
+const findByText = locator => element(by.text(locator))
+
+// DATA
+const USERNAME = {
+  tooShort: '12',
+  invalidCharacters: "username'",
+  taken: 'JS test 0',
+  valid: genUsername()
+}
+
+const PASSWORD = {
+  tooShort: 'yMvPLFupQ',
+  lacksNumber: 'yMvPLFupQj',
+  lacksUppercase: 'y768mv4plfupqjmu',
+  lacksLowercase: 'Y768MV4PLFUPQjMU',
+  mismatch: 'uMjQpuFLP4vM867y',
+  valid: 'y768Mv4PLFupQjMu'
+}
 
 beforeEach(async () => {
   await launchAppWithPermissions()
-  await navigateToHome()
+  // await navigateToHome()
 })
 
-describe.skip('Edge', () => {
-  it('should be able to create account', async () => {
-    // MATCHERS
-    const createAccountButton = element(by.text('Create an account'))
-    const getStartedButton = element(by.text('Get started'))
-    const usernameInput = element(by.type('RCTTextField'))
-    const nextButton = element(by.text('NEXT'))
-    const usernameTooShortError = element(by.text('Minimum 3 characters'))
-    const usernameInvalidCharactersError = element(by.text('Must only be ascii characters'))
+describe('Edge GUI: ', () => {
+  it('should be able to simply create an account', async () => {
+    // MATCHERS(Caon't be in before. Will be moved to POM)
+    const createAccountButton = element(by.text('Create account'))
```

Comment:
Create a scene folder and populate with files and functions that look like this...

const loginscene = () => ({
  createAccountButton: element(by.text('Create account')),
  getStartedButton: element(by.text('Get Started')),
  usernameInput: element(by.type('RCTUITextField')),
  nextButton: element(by.text('Next')),
  usernameTakenError: element(by.text('Username already exists')),
  passwordInput: element(by.type('RCTUITextField')).atIndex(0),
  confirmPasswordInput: element(by.type('RCTUITextField')).atIndex(1),
  passwordMismatchError: element(by.text('Does not match password')),
  pinInput: element(by.type('RCTUITextField')),
  confirmation1: element(by.type('RCTImageView')).atIndex(0),
  confirmation2: element(by.type('RCTImageView')).atIndex(1),
  confirmation3: element(by.type('RCTImageView')).atIndex(2),
  confirmation4: element(by.type('RCTImageView')).atIndex(3),
  confirmFinishButton: element(by.text('Confirm & Finish'))
})